core/models.py

```diff
@@ -1,3 +1,60 @@
 from django.db import models
 
-# Create your models here.
+
+class Deck(models.Model):
+    name = models.CharField(max_length=100, verbose_name="name of deck")
+    description = models.CharField(max_length=1200, verbose_name="Description of deck")
+    tag = models.CharField(max_length=50, verbose_name="tag")
```

Comment:
https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.ManyToManyField.through
Please add many to many field with trough field corresponded to DeckFlashCard

Reply:
i added but I have problem
AttributeError at /admin/core/deckflashcard/add/
'DeckFlashCard' object has no attribute 'deck'

Can you test create deck on your computer?

Reply:
on admin panel

Reply:
Fixed